src/components/radio-tile/radio-tile.spec.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import RadioButtonMapper from '../../__experimental__/components/radio-button/radio-button-mapper.component';
+import { RadioTile, RadioTileGroup } from '.';
+import { baseTheme } from '../../style/themes';
+
+// import Fieldset from '../fieldset';
+import {
+  StyledRadioTileFieldset,
+
+  StyledRadioTileContainer,
+  StyledRadioTile,
+  StyledRadioTileInput,
+  StyledDeselectButton
+} from './radio-tile.style';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+
+const radioValues = ['val1', 'val2', 'val3'];
+
+describe('RadioTile', () => {
+  let wrapper;
+
+  const render = (props) => {
+    wrapper = mount(
+      <RadioTile { ...props } />
+    );
+  };
+
+  beforeEach(() => {
+    render();
+  });
+
+  it('clicking on RadioTile invokes passed onChange callback', () => {
```

Comment:
It seems that this test is not executing click therefore onChange should not be triggering.